src/Action/CmdLine.hs

```diff
@@ -55,6 +55,7 @@ data CmdLine
         ,https :: Bool
         ,cert :: FilePath
         ,key :: FilePath
+        ,resrc :: Maybe FilePath
```

Comment:
Is `--resrc` the right flag? I think I'd prefer `--datadir`, as that's the concept Haskell programmers will expect.

Reply:
Whichever you'd prefer. I believe `resrc` was the name of a similar flag in Hoogle 4